devel/swiftformat/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        nicklockwood swiftformat 0.54.3
+github.tarball_from archive
+revision            0
+
+platforms           {darwin >= 21}
+categories          devel
+license             MIT
+maintainers         openmaintainer
```

Comment:
This I guess should be `nomaintainer`? What's the rule for new ports, usually I've only ever seen us add `nomaintainer` to old ports.

Reply:
you can submit a port with oh t a maintainer, but given the that you presumably care about something you submit it would be great if the submitter would volunteer to maintain it.

But indeed, "openmaintainer" is incorrect as that only signifies that others -besides the listed maintainer- can make changes without requiring approval.

Reply:
Thanks for the feedback. Fixed!